File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    answer = ''
    flag = True
    for i in s:
        if i ==' ':
            flag = True
        else:
            if flag == True:
                if 'a'<=i<='z':
                    i = i.upper()

            else:
                if 'A'<=i<='Z':
                    i = i.lower()
            flag = False

        answer += i

    return answer

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.cpp
#include <string>
#include <vector>
#include <cctype>
using namespace std;

string solution(string s) {
    string answer = "";
    bool first = true;
    for(auto now : s){
        if(first){
            if(now >= 'a' && now<='z'){
                answer += toupper(now);
            }else answer += now;
            first = false;
            if(now == ' '){
                first = true;
            }
        }
        else{
            if(now == ' '){
                first = true;
                answer += now; 
            }
            else{
                if(now>='A' && now<='Z'){
                    answer += tolower(now);
                }else answer += now;
            }
        }
        
    }
    return answer;
}